src/idp_fraud_event.py

```diff
@@ -0,0 +1,23 @@
+class IdpFraudEvent(object):
```

Comment:
Doesn't matter particularly, but you might be interested in https://docs.python.org/3/library/dataclasses.html if you didn't see it already.

Reply:
👍